src/main/java/de/doubleslash/keeptime/common/ColorTimeLine.java

```diff
@@ -0,0 +1,42 @@
+package de.doubleslash.keeptime.common;
```

Comment:
this should be, like the LicenseTableRow class just in the UI package - as this is just custom UI related